fighters/lucas/src/status.rs

```diff
@@ -2,12 +2,259 @@ use super::*;
 use globals::*;
 utils::import!(common::djc::attack_air_main_status);
 // status script import
- 
+
 pub fn install() {
     install_status_scripts!(
-        attack_air,
-        special_hi_attack
+        attack_air, 
+        special_hi_attack, 
+        //lucas_special_s_pre,
+        lucas_special_n_hold_main, // notably, do not try to install the main loop
+        //lucas_special_n_end
+        //lucas_special_s_main
     );
+
+    install_agent_resets!(lucas_reset);
+}
+
+#[fighter_reset]
+fn lucas_reset(fighter: &mut L2CFighterCommon) {
+    unsafe {
+        if fighter.kind() == *FIGHTER_KIND_LUCAS {
+            let charge_time = ParamModule::get_int(fighter.object(), ParamType::Agent, "attack_up_charge_time");
+            VarModule::set_int(fighter.object(), charge_time, vars::lucas::instance::SPECIAL_N_OFFENSE_UP_CHARGE_LEVEL);
+            VarModule::off_flag(fighter.object(), vars::lucas::instance::SPECIAL_N_OFFENSE_UP_ACTIVE);
+            VarModule::off_flag(fighter.object(), vars::lucas::instance::SPECIAL_N_OFFENSE_UP_INIT);
+            VarModule::off_flag(fighter.object(), vars::lucas::instance::SPECIAL_N_OFFENSE_UP_RELEASE_AFTER_WHIFF);
+        }
+    }
+}
+
```

Comment:
pls comment `// FIGHTER_LUCAS_STATUS_KIND_SPECIAL_N_HOLD //` here, to keep consistent with our status script organization